src/benchmark/trulens/benchmark/benchmark_frameworks/experiments/competitive_analysis.ipynb

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #61.                score = 0  # if there is an error, we assume the score is 0
</pre><p>If there is an error, we should skip this row instead of making the score 0? While running these tests, you could hit issues like rate limiting error, and that could impact benchmarks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>